packages/edgeidentity/README.md

```diff
@@ -94,18 +97,201 @@ public class MainApplication extends Application implements ReactApplication {
 ```
 ### [Identity for Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network)
 
-#### Importing the extension:
+### Importing the extension:
+In your React Native application, import the AEPEdgeIdentity extension as follows:
 ```javascript
 import {AEPIdentity} from '@adobe/react-native-aepedgeidentity';
 ```
 
-#### Getting the extension version:
+## API reference
+### Extension version:
+Returns the version of the AEPEdgeIdentity for Edge Network extension
+
+**Syntax**
 
+```javascript
+static extensionVersion(): Promise<string>
+```
+**Example**
 ```javascript
 AEPIdentity.extensionVersion().then(version => console.log("AdobeExperienceSDK: AEPEdgeIdentity version: " + version));
 ```
-#### Get Experience Cloud ID:
+### Get Experience Cloud ID:
```

Comment:
For these API reference headers can we use the API name and order them alphabetically? e.g. getExperienceCloudId , getIdentities ..